src/garage/torch/algos/ddpg.py

```diff
@@ -136,17 +137,18 @@ def train_once(self, itr, paths):
         ])
 
         last_average_return = np.mean(self.episode_rewards)
-        for train_itr in range(self.n_train_steps):
+        for _ in range(self.n_train_steps):
             if self.replay_buffer.n_transitions_stored >= self.min_buffer_size:  # noqa: E501
                 self.evaluate = True
                 samples = self.replay_buffer.sample(self.buffer_batch_size)
-                qf_loss, y, q, policy_loss = torch_to_np(
+                # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
the static analyzer can't figure out how long the return tuple is supposed to be.

i think this might be avoided more robustly by updating the implementation of `torch_to_np` to:

```python
return (v.numpy() for v in value_in)
```

or even
```python
return map(lambda v: v.numpy(), value_in)
```